packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -452,6 +430,28 @@ class DebuggerScreen extends Screen {
       sizes: [80, 20],
       minSize: [200, 60],
     );
+
+    final options = <String, dynamic>{
+      'mode': 'dart',
+      'lineNumbers': true,
+      'gutters': <String>['breakpoints'],
+    };
+    final CodeMirror codeMirror =
+    CodeMirror.fromElement(_sourceArea.element, options: options);
```

Comment:
nit: dartfmt

Reply:
I wish copy and paste in intellij didn't mess up indentation. :(